creator-node/src/AsyncProcessingQueue.js

```diff
@@ -50,47 +50,7 @@ class AsyncProcessingQueue {
     this.libs = libs
 
     this.queue.process(MAX_CONCURRENCY, async (job, done) => {
-      const { logContext, task } = job.data
-
-      const func = this.getFn(task)
```

Comment:
hard to tell from gitdiff but there is no diff here - its just moving same code into function

Reply:
yup thats exactly it

Reply:
ye was just commenting for clarity